lib/workos/audit_logs.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+# typed: true
+
+require 'net/http'
+require 'uri'
+
+module WorkOS
+  # The Audit Logs module provides convenience methods for working with the
+  # WorkOS Audit Logs platform. You'll need a valid API key.
+  module AuditLogs
+    class << self
+      extend T::Sig
+      include Client
+
+      # Create an Audit Log Event.
+      #
+      # @param [String] organization An Organization ID
+      # @param [Hash] event An Audit Log Event
+      # @param [String] idempotency_key An idempotency key
+      #
+      # @return [nil]
+      sig do
+        params(
+          organization: String,
+          event: Hash,
+          idempotency_key: T.nilable(String),
+        ).returns(::T.untyped)
```

Comment:
Should this be `void` instead of returning `T.untyped`?

```suggestion
        ).void
```

Reply:
I updated the related assertions to `expect(response).to eq T::Private::Types::Void::VOID` which seems iffy, not sure if there is a more standard way of asserting a void response.